pkg/git/client.go

```diff
@@ -252,6 +252,11 @@ func (c *client) envsForRepo(remote string) []string {
 	return append(envs, c.gitEnvs...)
 }
 
+// RunGitCommand runs a git command with the given arguments.
```

Comment:
we don't use this anymore 👀

Reply:
Thank you. I removed it on this commit
https://github.com/pipe-cd/pipecd/pull/5020/commits/355ce05ef71a28865fb680c31fe98ac0a6a5556d